### ISSUE TITLE
Not posible to modify legend position 

### ISSUE THREAD

**Issue by set92** [pyviz/hvplot#144]
#119 technically allows to change the position of the legend. But I tried it and it returns an error "ValueError: Boolean 'show_legend' only takes a Boolean value.". I checked the pull code and is another error.

Also, it is possible to have a button to hide the legend on the right? in the same place as the save button. Because having a plot with 17 elements means having a long legend and is overlapping with the data, so I thought if it would be possible to hide it after muting some elements, or make it smaller.

**Input**
```
mvtos_dict['13017094'].hvplot(x = 'Fecha', y = ['Stock'], legend='left', width=1700, height=850, grid=True, label=f'13017094 - {necesidad_104[13017094]}') *
mvtos_dict['13037094'].hvplot(x = 'Fecha', y = ['Stock'], label=f'13037094 - {necesidad_104[13037094]}') *
.... (15 more rows with different keys of the dict)
```

**Output**

> ValueError                                Traceback (most recent call last)
> ~\Anaconda3\envs\py36\lib\site-packages\IPython\core\formatters.py in __call__(self, obj, include, exclude)
>     968 
>     969             if method is not None:
> --> 970                 return method(include=include, exclude=exclude)
>     971             return None
>     972         else:
> 
> ~\Anaconda3\envs\py36\lib\site-packages\holoviews\core\dimension.py in _repr_mimebundle_(self, include, exclude)
>    1391         combined and returned.
>    1392         """
> -> 1393         return Store.render(self)
>    1394 
>    1395 
> 
> ~\Anaconda3\envs\py36\lib\site-packages\holoviews\core\options.py in render(cls, obj)
>    1451         data, metadata = {}, {}
>    1452         for hook in hooks:
> -> 1453             ret = hook(obj)
>    1454             if ret is None:
>    1455                 continue
> 
> ~\Anaconda3\envs\py36\lib\site-packages\holoviews\ipython\display_hooks.py in pprint_display(obj)
>     270     if not ip.display_formatter.formatters['text/plain'].pprint:
>     271         return None
> --> 272     return display(obj, raw_output=True)
>     273 
>     274 
> 
> ~\Anaconda3\envs\py36\lib\site-packages\holoviews\ipython\display_hooks.py in display(obj, raw_output, **kwargs)
>     240     elif isinstance(obj, (CompositeOverlay, ViewableElement)):
>     241         with option_state(obj):
> --> 242             output = element_display(obj)
>     243     elif isinstance(obj, (Layout, NdLayout, AdjointLayout)):
>     244         with option_state(obj):
> 
> ~\Anaconda3\envs\py36\lib\site-packages\holoviews\ipython\display_hooks.py in wrapped(element)
>     140         try:
>     141             max_frames = OutputSettings.options['max_frames']
> --> 142             mimebundle = fn(element, max_frames=max_frames)
>     143             if mimebundle is None:
>     144                 return {}, {}
> 
> ~\Anaconda3\envs\py36\lib\site-packages\holoviews\ipython\display_hooks.py in element_display(element, max_frames)
>     186         return None
>     187 
> --> 188     return render(element)
>     189 
>     190 
> 
> ~\Anaconda3\envs\py36\lib\site-packages\holoviews\ipython\display_hooks.py in render(obj, **kwargs)
>      63         renderer = renderer.instance(fig='png')
>      64 
> ---> 65     return renderer.components(obj, **kwargs)
>      66 
>      67 
> 
> ~\Anaconda3\envs\py36\lib\site-packages\holoviews\plotting\bokeh\renderer.py in components(self, obj, fmt, comm, **kwargs)
>     247         # Bokeh has to handle comms directly in <0.12.15
>     248         comm = False if bokeh_version < '0.12.15' else comm
> --> 249         return super(BokehRenderer, self).components(obj,fmt, comm, **kwargs)
>     250 
>     251 
> 
> ~\Anaconda3\envs\py36\lib\site-packages\holoviews\plotting\renderer.py in components(self, obj, fmt, comm, **kwargs)
>     319             plot = obj
>     320         else:
> --> 321             plot, fmt = self._validate(obj, fmt)
>     322 
>     323         widget_id = None
> 
> ~\Anaconda3\envs\py36\lib\site-packages\holoviews\plotting\renderer.py in _validate(self, obj, fmt, **kwargs)
>     218         if isinstance(obj, tuple(self.widgets.values())):
>     219             return obj, 'html'
> --> 220         plot = self.get_plot(obj, renderer=self, **kwargs)
>     221 
>     222         fig_formats = self.mode_formats['fig'][self.mode]
> 
> ~\Anaconda3\envs\py36\lib\site-packages\holoviews\plotting\bokeh\renderer.py in get_plot(self_or_cls, obj, doc, renderer, **kwargs)
>     132             curdoc().theme = self_or_cls.theme
>     133         doc.theme = self_or_cls.theme
> --> 134         plot = super(BokehRenderer, self_or_cls).get_plot(obj, renderer, **kwargs)
>     135         plot.document = doc
>     136         return plot
> 
> ~\Anaconda3\envs\py36\lib\site-packages\holoviews\plotting\renderer.py in get_plot(self_or_cls, obj, renderer, **kwargs)
>     201                              **kwargs)
>     202             plot = self_or_cls.plotting_class(obj)(obj, renderer=renderer,
> --> 203                                                    **plot_opts)
>     204             defaults = [kd.default for kd in plot.dimensions]
>     205             init_key = tuple(v if d is None else d for v, d in
> 
> ~\Anaconda3\envs\py36\lib\site-packages\holoviews\plotting\bokeh\element.py in __init__(self, overlay, **params)
>    1624 
>    1625     def __init__(self, overlay, **params):
> -> 1626         super(OverlayPlot, self).__init__(overlay, **params)
>    1627         self.set_root(params.pop('root', None))
>    1628 
> 
> ~\Anaconda3\envs\py36\lib\site-packages\holoviews\plotting\plot.py in __init__(self, overlay, ranges, batched, keys, group_counter, **params)
>    1143         self.cyclic_index_lookup = {}
>    1144         self.zoffset = 0
> -> 1145         self.subplots = self._create_subplots(ranges)
>    1146         self.traverse(lambda x: setattr(x, 'comm', self.comm))
>    1147         self.top_level = keys is None
> 
> ~\Anaconda3\envs\py36\lib\site-packages\holoviews\plotting\plot.py in _create_subplots(self, ranges)
>    1206         subplots = OrderedDict()
>    1207         for (key, vmap, streams) in zip(keys, vmaps, dmap_streams):
> -> 1208             subplot = self._create_subplot(key, vmap, streams, ranges)
>    1209             if subplot is None:
>    1210                 continue
> 
> ~\Anaconda3\envs\py36\lib\site-packages\holoviews\plotting\plot.py in _create_subplot(self, key, obj, streams, ranges)
>    1290                         projection=self.projection,
>    1291                         zorder=zorder, **passed_handles)
> -> 1292         return plottype(obj, **plotopts)
>    1293 
>    1294 
> 
> ~\Anaconda3\envs\py36\lib\site-packages\holoviews\plotting\bokeh\element.py in __init__(self, element, plot, **params)
>     140     def __init__(self, element, plot=None, **params):
>     141         self.current_ranges = None
> --> 142         super(ElementPlot, self).__init__(element, **params)
>     143         self.handles = {} if plot is None else self.handles['plot']
>     144         self.static = len(self.hmap) == 1 and len(self.keys) == len(self.hmap)
> 
> ~\Anaconda3\envs\py36\lib\site-packages\holoviews\plotting\bokeh\plot.py in __init__(self, *args, **params)
>     139     def __init__(self, *args, **params):
>     140         root = params.pop('root', None)
> --> 141         super(BokehPlot, self).__init__(*args, **params)
>     142         self._document = None
>     143         self._root = root
> 
> ~\Anaconda3\envs\py36\lib\site-packages\holoviews\plotting\plot.py in __init__(self, element, keys, ranges, dimensions, batched, overlaid, cyclic_index, zorder, style, overlay_dims, stream_sources, streams, **params)
>     828         super(GenericElementPlot, self).__init__(keys=keys, dimensions=dimensions,
>     829                                                  dynamic=dynamic,
> --> 830                                                  **dict(params, **plot_opts))
>     831         self.streams = get_nested_streams(self.hmap) if streams is None else streams
>     832         if self.top_level:
> 
> ~\Anaconda3\envs\py36\lib\site-packages\holoviews\plotting\plot.py in __init__(self, keys, dimensions, layout_dimensions, uniform, subplot, adjoined, layout_num, style, subplots, dynamic, renderer, comm, **params)
>     247         params = {k: v for k, v in params.items()
>     248                   if k in self.params()}
> --> 249         super(DimensionedPlot, self).__init__(**params)
>     250 
>     251 
> 
> ~\Anaconda3\envs\py36\lib\site-packages\param\parameterized.py in __init__(self, **params)
>    1907         self.param._generate_name()
>    1908 
> -> 1909         self.param._setup_params(**params)
>    1910         object_count += 1
>    1911 
> 
> ~\Anaconda3\envs\py36\lib\site-packages\param\parameterized.py in override_initialization(self_, *args, **kw)
>     704         original_initialized=parameterized_instance.initialized
>     705         parameterized_instance.initialized=False
> --> 706         fn(parameterized_instance,*args,**kw)
>     707         parameterized_instance.initialized=original_initialized
>     708     return override_initialization
> 
> ~\Anaconda3\envs\py36\lib\site-packages\param\parameterized.py in _setup_params(self_, **params)
>     846                 self.param.warning("Setting non-parameter attribute %s=%s using a mechanism intended only for parameters",name,val)
>     847             # i.e. if not desc it's setting an attribute in __dict__, not a Parameter
> --> 848             setattr(self,name,val)
>     849 
>     850     @classmethod
> 
> ~\Anaconda3\envs\py36\lib\site-packages\param\__init__.py in __set__(self, obj, val)
>     831         else:
>     832             if not isinstance(val,bool):
> --> 833                 raise ValueError("Boolean '%s' only takes a Boolean value."%self._attrib_name)
>     834 
>     835             if val is not True and val is not False:
> 
> ValueError: Boolean 'show_legend' only takes a Boolean value.
>

**Comment by philippjfr** [comment]
@set92 Could you report your version of hvplot? I can't reproduce the error, but it does seem to ignore the legend position in certain cases.

**Comment by LinuxIsCool** [comment]
Was getting the same problem on version 0.2.1.
Upgraded to 0.3.0. Not getting an error anymore, but the parameter is having no effect.

**Comment by philippjfr** [comment]
I'll try to fix this today and then tag 0.3.1 asap.
